internal/controllers/addon/phase_ensure_subscription_test.go

```diff
@@ -108,3 +109,127 @@ func assertUnreconciledSubscription(t *testing.T, sub *operatorsv1alpha1.Subscri
 	assert.Error(t, err)
 	assert.EqualError(t, err, controllers.ErrNotOwnedByUs.Error())
 }
+
+func TestCreateSubscriptionConfigObject(t *testing.T) {
+	testCases := []struct {
+		commonInstallOptions addonsv1alpha1.AddonInstallOLMCommon
```

Comment:
@Ankit152 minor,can you please name this commonOLMInstallOptions?

Reply:
Done